VAMobile/src/screens/HealthScreen/Pharmacy/RefillScreens/RefillRequestSummary.test.tsx

```diff
@@ -23,17 +18,9 @@ context('RefillRequestSummary', () => {
         ...prescriptionState,
       },
     }
-    component = render(<RefillRequestSummary {...props} />, { preloadedState: store })
-    testInstance = component.UNSAFE_root
+    render(<RefillRequestSummary {...props} />, { preloadedState: store })
   }
 
-  it('initializes correctly', async () => {
-    initializeTestInstance()
-    await waitFor(() => {
-      expect(component).toBeTruthy()
-    })
-  })
-
   describe('when all request submit successfully', () => {
     it('should display successful summary', async () => {
       await waitFor(() => {
```

Comment:
I don't think this needs to be wrapped in a `waitFor`. Same for the other one in this file

Reply:
removed